pkg/configtx/constants.go

```diff
@@ -6,37 +6,35 @@ SPDX-License-Identifier: Apache-2.0
 
 package configtx
 
-import "math"
-
 const (
 	// These values are fixed for the genesis block.
 	msgVersion = 0
 	epoch      = 0
 
-	defaultHashingAlgorithm = "SHA256"
-
-	defaultBlockDataHashingStructureWidth = math.MaxUint32
-
-	// ConsortiumKey is the key for the common.ConfigValue for the Consortium message.
+	// ConsortiumKey is the key for the ConfigValue of a
+	// Consortium.
 	ConsortiumKey = "Consortium"
 
-	// HashingAlgorithmKey is the common.ConfigValue type key name for the HashingAlgorithm message.
+	// HashingAlgorithmKey is the key for the ConfigValue of a
+	// HashingAlgorithm.
 	HashingAlgorithmKey = "HashingAlgorithm"
 
-	// BlockDataHashingStructureKey is the common.ConfigValue type key name for the BlockDataHashingStructure message.
+	// BlockDataHashingStructureKey is the key for the ConfigValue
+	// of a BlockDataHashingStructure.
 	BlockDataHashingStructureKey = "BlockDataHashingStructure"
 
-	// OrdererAddressesKey is the common.ConfigValue type key name for the OrdererAddresses message.
+	// OrdererAddressesKey is the key for the ConfigValue, OrdererAddresses.
 	OrdererAddressesKey = "OrdererAddresses"
 
-	// CapabilitiesKey is the name of the key which refers to capabilities, it appears at the channel,
-	// application, and orderer levels and this constant is used for all three.
+	// CapabilitiesKey is the key for the ConfigValue, capabilities.
+	// CapabiltiesKey can be used at the channel, application, and orderer levels.
 	CapabilitiesKey = "Capabilities"
 
-	// EndpointsKey is the common.ConfigValue key name for the Endpoints message in the OrdererOrgGroup.
+	// EndpointsKey is the key for the ConfigValue, Endpoints in
+	// a OrdererOrgGroup.
 	EndpointsKey = "Endpoints"
 
-	// MSPKey is the key for the MSP definition in orderer groups.
+	// MSPKey is the key for the ConfigValue MSP in orderer ConfigGroups.
```

Comment:
Can you check if this is only used for Orderer? I feel like this is used for any org ConfigGroup

Reply:
I know it can also be used for application at least.

Reply:
MSP configuration is for an org in general. Off the top of my head I think this key is used for the MSP config value in which case it would apply to all orgs